lib/common/zstd_internal.h

```diff
@@ -137,7 +137,10 @@ static const U32 OF_defaultNormLog = 5;
 /*-*******************************************
 *  Shared functions to include for inlining
 *********************************************/
-static void ZSTD_copy8(void* dst, const void* src) { memcpy(dst, src, 8); }
+static void ZSTD_copy8(void* dst, const void* src)
+{
+    MEM_write64(dst, MEM_read64(src));
```

Comment:
Yes, I feel confident.
I also don't consider `memcpy()` to be disabled :
it's just displaced by some more efficient method (under conditions) for the specific usage of `MEM_read32()` and equivalents.
But `memcpy()` is still used in many other places in the code, so it's all but disabled.